operations/research-and-development/engineering/cloud-infrastructure-cost-kpis.md

```diff
@@ -0,0 +1,77 @@
+# Cloud Infrastructure Cost KPIs
+
+The Cloud SRE team uses certain KPIs to monitor cloud infrastructure costs from the engineering standpoint. These KPIs are used to help the business and the Cloud SRE team to set goals for cost optimization.
+
+As of July 2021, the goals for the SRE team are to
+
+ - Decrease the Average Cost Per Workspace (APWC)
+ - Keep development and test environments costs (DM) below 20% of total costs
+ - Increase the WB fraction (ratio of paid workspaces relative to total workspaces)
+
+Below are the primary and seconday KPIs used to measure each:
+
+## Primary KPIs
+
+### Average Production Workspace Cost
+
+ - **Description**: The fraction which compares the amount of Production Environment costs with the number of workspaces which at a given month of the Cloud Infrastructure service.
+ - **Formula**: APWC = Production Environment Costs/Number of Workspaces  
+
+### Average Production Freemium Cost
+
+ - **Description**: Roughly is how much free workspaces cost to us. The fraction which compares the amount of Production Environment costs with the number of active free workspaces.
+ - **Formula**: APFC = Production Environment Costs/Freemium Active Workspaces
+Note: Until February 2021 we measure all workspaces as active since we introduced hibernation functionality the last 3 days of February
+
+### Average Subscription Workspace Cost
+
+ - **Description**: The fraction which compares the amount of Production Environment costs with the number of workspaces that have 11 or more users which at a given month the Cloud Infrastructure service.
+ - **Formula**: ASWC = Production Environment Costs/ Paid Workspaces
```

Comment:
@jasonblais Also here we need clarification. In my opinion as paid workspaces should consider the workspaces that successfully paid an invoice for the month we measure (previous month).

Reply:
Thanks! Added, let me know your thoughts